lte/gateway/c/core/oai/tasks/s1ap/s1ap_mme.c

```diff
@@ -219,10 +219,12 @@ static int handle_message(zloop_t* loop, zsock_t* reader, void* arg) {
     } break;
 
     case MME_APP_CONNECTION_ESTABLISHMENT_CNF: {
-      is_task_state_same = true;  // the following handler does not modify state
-      is_ue_state_same   = true;
+      is_task_state_same =
+          false;  // the following handler does not modify state
```

Comment:
is the comment still valid here?

Reply:
yes, it's still valid on v1.6

Reply:
hmm, the comment did not look correct on master either. if the state does not remain the same, then the handler should be modifying the state, no?

Reply:
you're right, the comment was not updated on master as well, i will update it here for v1.6 and on master on a separate PR